manifests/transport_https.pp

```diff
@@ -0,0 +1,8 @@
+# Allow APT to use repos served over HTTPS
+class apt::transport_https (
+  $ensure = 'present'
+) {
+  package { ['apt-transport-https', 'ca-certificates']:
```

Comment:
I recommend using stdlib's `ensure_packages` function, you can prevent duplicate resource errors on accidental includes or if another class has already installed these packages.